### PR TITLE
Added headline,blurbs and partners section as per v3 figma design

### PR DESCRIPTION
**Pull request by siddhantsingh186**
## Related Issue

- Add the headline, blurbs, and partners section on landing page

Closes: #289 

### Describe the changes you've made

Implemented the headlines, blurbs, and partners section as per the v3 design.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

Tested the changes locally with bright mode.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

image
image

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -49,21 +44,175 @@ export default function IndexPage() {
...
+            <div className="flex items-center">
+              <Icons.discord className="h-5 w-5 mr-1 fill-current cursor-pointer" />
+              <span className="cursor-pointer">Discord</span>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      {/* Blurbs */}
+      <section className="flex py-0 px-40">
```
Comment (mkubdev) [inline app/page.tsx:103]:
```suggestion
      <section className="flex justify-center py-0 px-40">
```

You can add `justify-center` to center the section. :)

**Review by mkubdev — CHANGES_REQUESTED** [review]
Hello, thanks a lot for your contribution 🎉🤝
Everything looks good to me, kindly ask you to add a centering class!

> I renamed js attributes like `stroke-linecap` into `strokeLinecap` because[ jsx use camelCase](https://reactjs-bot.github.io/react/docs/introducing-jsx.html) 💡

File: app/page.tsx (hunk 1 above)
Comment (siddhantsingh186) [reply app/page.tsx]: Thanks for the suggestion. I have added the suggested changes.

**Review by siddhi-244 — APPROVED** [review]
Looks awesome!

**Review by Mustafiz04 — APPROVED** [review]
The changes look good.

**Review by vinzvinci — APPROVED** [review]
LGTM!

**Comment by mkubdev** [comment]
Awesome 🚀

**Pull request merged**